masquerade_tool/templates/masquerade_tool/index.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'masquerade_tool/base.html' %}
+{% load static %}
+
+{% block breadcrumb %}
+  <nav>
+    <h3>
+      <a href="{% url 'dashboard_account' %}">Admin Console</a>
+      <small><i class="fa fa-chevron-right"></i></small>
+      Masquerade Tool
+    </h3>
+  </nav>
+{% endblock %}
+
+{% block content %}
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-med-9">
+        <br><br>
+        <p class="lead"><strong>Click the button below to temporarily assume the Masquerade role.</strong></p>
+        <p class="lead">Once granted, you'll be able to use
+          Canvas'
+          "<a href="https://community.canvaslms.com/t5/Admin-Guide/How-do-I-act-as-another-user-in-an-account/ta-p/161"
```

Comment:
We may want to add `rel="noreferrer"` to this link, per best practices on cross-site links: https://web.dev/external-anchors-use-rel-noopener/